index.html

```diff
@@ -46,6 +46,42 @@ <h1>I’m Kaweesi Glad to see you!</h1>
           </li>
         </ul>
       </section>
+      <div class="wrapper2">
+        <section class="tonic">
+          <div class="imgez"><img src="./images/snap.png" alt="snap" /></div>
+          <div>
+            <h2>Tonic</h2>
+          </div>
+  
+          <div>
+            <ul id="profs">
+              <li id="cannopy">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
```

Comment:
Please, add the button interactions to match the given design. Sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit right now. You'll get more prepared for a typical workplace experience where you might be required to reproduce a given design as strictly as possible.

Follow the original vision if it's pretty and works, it will be a wonderful website.

Take a look at the interactions recommended by the design:

![image](https://user-images.githubusercontent.com/20554583/185735589-d08f9075-44ac-4443-b105-3fd024093a3b.png)

Currently, your buttons have none of them.